app/views/content/train-to-be-a-teacher/what-is-qts.md

```diff
@@ -64,7 +64,7 @@ $qualified-outside-the-uk$
 If your teacher training course leads to QTS: 
 
 * your course will be regulated by the Department for Education (DfE), ensuring high quality training 
-* you could be eligible for a scholarship or bursary to help you train
+* you could be eligible for [a scholarship or bursary to help you train](/funding-and-support/scholarships-and-bursaries)
```

Comment:
Tiny point - should this be consistent with the link on the other page where the hyperlink text is: eligible for a scholarship or bursary?